build.sbt

```diff
@@ -1,77 +1,107 @@
+import Dependencies.Library
```

Comment:
minor: The imports are usually after the header about the license.

Reply:
I did not do that myself, this is the result of auto-formatting

Reply:
Still, I only see it in this file. I am not saying that it's the wrong or correct way it's just inconsistent.